.gitignore

```diff
@@ -17,3 +17,4 @@ npm-debug.log
 testem.log
 
 config/*.yml
+.idea/
```

Comment:
I have no problem adding this, but you might want to look into setting up a global .gitignore: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore